pyClient/zeth/mimc.py

```diff
@@ -0,0 +1,135 @@
+from zeth.constants import ZETH_PRIME
+from Crypto.Hash import keccak
+from typing import Any, Tuple, Iterable, cast
+
+
+def keccak_256(data: bytes) -> keccak.Keccak_Hash:
+    return keccak.new(data, digest_bits=256)
+
+
+class MiMC7:
+    def __init__(
+            self,
+            seed: str = "clearmatics_mt_seed",
```

Comment:
Seems like this seed should be added to the constants file (named something like `MIMC_MT_SEED`)

Reply:
Updated this and moved all the test code out of here into the `test` module.